powerline.el

```diff
@@ -467,9 +468,13 @@ static char * %s[] = {
    "\\`[ \t\n\r]+" ""
    (replace-regexp-in-string "[ \t\n\r]+\\'" "" s)))
 
+(defun strip-text-properties (txt)
```

Comment:
unless you use `strip-text-properties` somewhere else then you should just inline it. also it would need to be called `powerline--strip-text-properties` because that's namespacing in elisp :/